test_regress/t/t_assert_ctl_arg_unsup.out

```diff
@@ -0,0 +1,18 @@
+%Error-UNSUPPORTED: t/t_assert_ctl_arg_unsup.v:15:7: Unsupported: assert control assertion_type
+                                                   : ... note: In instance 't'
+   15 |       $assertcontrol(OFF, EXPECT);
+      |       ^~~~~~~~~~~~~~
```

Comment:
Please change error message to point to the UNIQUE, as that is the type.

Reply:
When I changed the error message to point to the incorrect argument; `nodep->assertTypesp()->v3warn(...)` it points to the variable definition and the error message looks like this:
```
+%Error-UNSUPPORTED: t/t_assert_ctl_arg_unsup.v:12:19: Unsupported: assert control assertion type
+                                                    : ... note: In instance 't'
+   12 |    let PRIORITY = 128;
+      |                   ^~~
```

Is there any way to force error FileLine to point to the second argument?

Reply:
You're right, we'll need to stick with pointing at the assert instead.